source/common/listener_manager/listener_impl.cc

```diff
@@ -72,6 +73,20 @@ bool shouldBindToPort(const envoy::config::listener::v3::Listener& config) {
 }
 } // namespace
 
+FcdsApiPtr ListenerImpl::createFcdsSubscription(
+    const envoy::config::listener::v3::Listener::FcdsConfig& fcds_config) {
+  std::unique_ptr<FcdsApiImpl> subscription =
+      std::make_unique<FcdsApiImpl>(fcds_config.config_source(), fcds_config.resources_locator(),
+                                    name_, parent_.server_.clusterManager(), listenerScope(),
+                                    *dynamic_init_manager_, parent_, validation_visitor_);
+
+  if (!fcds_config.start_listener_without_warming()) {
```

Comment:
Agree with @adisuissa to move this to another PR. I don't understand the use case for early start.